gatling-jms/src/main/scala/io/gatling/jms/request/JmsRequestBuilder.scala

```diff
@@ -24,29 +24,30 @@ import io.gatling.jms._
 import io.gatling.jms.action.JmsReqReplyBuilder
 
 import com.softwaremill.quicklens._
+import io.gatling.jms.action.JmsRequestSendBuilder
 
 case class JmsRequestBuilderBase(requestName: String) {
 
   def reqreply(implicit configuration: GatlingConfiguration) = JmsRequestBuilderQueue(requestName, JmsReqReplyBuilder.apply(_, configuration))
+
+  def send(implicit configuration: GatlingConfiguration) = JmsRequestBuilderQueue(requestName, JmsRequestSendBuilder.apply(_, configuration))
```

Comment:
I think it's wrong to use a `JmsRequestBuilderQueue` as it will expose some methods that are specific to `reqreply`, such as a reply queue. We have to share code, but still, I think we need 2 different builders hierarchy.